src/toil/jobStores/abstractJobStore.py

```diff
@@ -108,6 +108,17 @@ def __init__(self, locator):
 class AbstractJobStore(with_metaclass(ABCMeta, object)):
     """
     Represents the physical storage for the jobs and files in a Toil workflow.
+    
+    JobStores are responsible for storing :class:`toil.job.JobDescription`s
```

Comment:
Thanks for adding this, this is great.

Reply:
The part about "the information from" was from a different design, so I am fixing that. `loadJob` wants the whole JobDescription. When I wrote this I had it taking just the command.